source/gameengine/Ketsji/KX_FontObject.h

```diff
@@ -45,6 +45,8 @@ class KX_FontObject : public KX_GameObject
 
 	virtual ~KX_FontObject();
 
+	MT_Vector2 GetTextSize();
```

Comment:
I'm a big fan of `const` :)
you could write:
```
const MT_Vector& Get…();
```

The function is not constant because some BLF operations in it are certainly not.